packages/components/menu/src/menu.tsx

```diff
@@ -71,10 +71,7 @@ function Menu<T extends object>(props: Props<T>, ref: ForwardedRef<HTMLUListElem
   );
 }
 
-export type MenuProps<T extends object = object> = Omit<
```

Comment:
I found type `Props<T>` don't have these types export so i change to Omit in the MenuItemVariantProps